app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -2251,6 +2251,118 @@ class PatientRepositoryAndroidTest {
         .inOrder()
   }
 
+  @Test
+  fun querying_all_patients_in_a_facility_should_set_the_the_latest_recorded_measurement_recorded_on_within_any_facility() {
```

Comment:
Can we also add another test that a patient shows up in the list of "All patients in facility" if they have only blood sugar measurements in a specific facility?

Reply:
I have added a test for All patients in the facility in the previous PR. Should I still add it?
`querying_all_patients_in_a_facility_should_return_only_patients_in_that_facility_ordered_by_name`

Reply:
Oh wait, I didn't check the entire file. Will check now.